apps/web/src/components/features/properties/PropertyDetail.tsx

```
@@ -0,0 +1,267 @@
+                  <input
+                    type="date"
+                    className="border-0 p-0 focus:outline-none w-full bg-transparent"
+                    placeholder="Add date"
+                  />
+                </div>
+              </div>
+            </div>
+
+            <div className="space-y-2 mb-6">
+              <label className="text-sm font-medium">Guests</label>
+              <div className="flex items-center border rounded-md p-2 bg-background">
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)